### PR TITLE
 feat: add the changelog action

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**

### CODE REVIEW

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -80,3 +84,15 @@ jobs:
...
+
+  changelog-fragment:
+    name: "Create changelog fragment"
+    needs: [labeler]
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: ansys/actions/doc-changelog@main
```
Comment (RobPasMue) [inline .github/workflows/label.yml:96]:
```suggestion
    - uses: ansys/actions/doc-changelog@v6
```

**Review by RobPasMue — COMMENTED** [review]
I'd recommend you also remove the *md files from the vale check =)

**Review by RobPasMue — APPROVED** [review]
LGTM!

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Pull request merged**